examples/plot_poisson_2.py

```diff
@@ -90,7 +91,14 @@
 ########################################################
 
 z = np.linspace(0., 10., 100)
-qu = glm_poissonexp.qu(z)
+
+eta = 4.0
+qu = deepcopy(z)
```

Comment:
maybe stupid question but why is `qu` always used as an abbreviation for "conditional intensity" ? same question for `lmb`. I couldn't figure it out yet